weldx/asdf/tags/weldx/core/common_types.py

```diff
@@ -135,4 +138,8 @@ def from_tree(cls, tree, ctx):
         if "unit" in tree:  # convert to pint.Quantity
             data = Q_(data, tree["unit"])
 
-        return Variable(tree["name"], tree["dimensions"], data)
+        # FIXME: for some reason, "attrs" is not set in the tree
+        # (sometimes, e.g. coordinate transformation schema)
+        attrs = tree.get("attrs", None)
```

Comment:
Haven't looked deeper into what you are actually doing, but so far, it shouldn't be a bug. WelDX filters unset values (`None`) from the tree so that they won't end up in the file.
```suggestion
        attrs = tree.get("attrs", None)
```